packages/prop-house-webapp/src/components/TruncateThousands/index.tsx

```diff
@@ -8,7 +8,12 @@ const TruncateThousands: React.FC<{ amount: number; decimals?: number }> = props
         : amount
       : Number(amount).toFixed(0);
 
-  return <>{addDecimals(amount > 1000 ? amount / 1000 : amount)}</>;
+  return (
+    <>
+      {addDecimals(amount >= 1000 ? amount / 1000 : amount)}
```

Comment:
nit: 
```      {addDecimals(amount >= 1000 ? `${amount} K` / 1000 : amount)}```
not sure if there's something wrong w that

Reply:
> nit: `` {addDecimals(amount >= 1000 ? `${amount} K` / 1000 : amount)} `` not sure if there's something wrong w that

yea, you cant perform a math operation on a string